Competition2017/src/competition/Robot.java

```diff
@@ -45,5 +47,9 @@ public void autonomousInit() {
         // Base implementation will run the command
         super.autonomousInit();
     }
-   
+    @Override
+    public void teleopInit(){
+        super.teleopInit();
+        drive.precisionMode.set(false);
```

Comment:
just because the robot could explode and we can't use final here, worth adding a check to make sure `drive` is not null.